addons/account/wizard/base_document_layout.py

```diff
@@ -25,28 +24,26 @@ def document_layout_save(self):
     @api.depends('qr_code')
     def _compute_preview(self):
         """ This override is needed to add the depends on the qr code and the move """
-        super()._compute_preview()
         if self.env.context.get('active_model') != 'account.move':
-            return
+            return super()._compute_preview()
         move = self.env['account.move'].browse(self.env.context.get('active_id'))
         # If there is a move in the context then, the value put in the preview come from the move
         if not move.exists():
-            return
+            return super()._compute_preview()
 
         styles = self._get_asset_style()
         for wizard in self:
             if wizard.report_layout_id:
-                preview_css, wizard_with_logo = wizard._get_render_information(styles)
-
                 # We don't want to display the qr_code twice since we decided to put a fake one.
                 move.display_qr_code = False
-                wizard.preview = wizard_with_logo.env['ir.ui.view']._render_template('account.report_invoice_wizard_iframe', {
-                    'company': wizard_with_logo,
-                    'preview_css': preview_css,
-                    'is_html_empty': is_html_empty,
-                    'o': move,
-                    'qr_code': wizard.qr_code,
-                })
+                wizard.preview = wizard.env['ir.ui.view']._render_template(
```

Comment:
That context is only used for `company` key in `_render_template` args not when we call it [see](https://github.com/odoo/odoo/pull/152270/files#diff-7643ad526c826f8b1976544be18452cb40dec3b4e462500282044c73818c12e6R143)
I did check if everything is working properly with or without that context it was only used here before because they had that data from that method
Here I removed it because it does not seem useful here :slightly_smiling_face: